myui/__init__.py

```diff
@@ -15,14 +19,37 @@ def __init__(self,handlers,settings):
 class BaseHandler(tornado.web.RequestHandler):
     pass
 
+def parse_log_file_option(option):
```

Comment:
This doesn't look necessary unless the TODO, "Get application log config working", was working.